receiver/dockerstatsreceiver/config.go

```diff
@@ -49,11 +49,8 @@ func (config Config) Validate() error {
 	if config.Endpoint == "" {
 		return errors.New("endpoint must be specified")
 	}
-	if config.CollectionInterval == 0 {
```

Comment:
removed since this is a redundant check already covered by `ScraperControllerSettings` validation.